wbooks/src/Books.js

```diff
@@ -1,37 +1,35 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 import myBooks from './books.json';
 import defaultBookIcon from './default_book.svg'
 import './Books.css';
 
 class Books extends Component {
-  constructor(props) {
-    super(props);
-  }
 
   filteredBooks = () => {
-    if (this.props.filterText == '' || this.props.filterType == '') {
+    if (this.props.filterText === '' || this.props.filterType === '') {
       return myBooks;
     }
-
     return myBooks.filter(book => {
-      console.log((this.props.filterText));
       return book[this.props.filterType].toUpperCase().includes(this.props.filterText.toUpperCase());
     });
   }
 
   render(){
      return (
-       <div className="body">
+       <div className="grid">
         {this.filteredBooks().map(item =>
-          <div className="book">
-          {(item.image_url == null) ? (
-            <img className="default-book-icon" src={defaultBookIcon} />
-          ) : (
-            <img className="book-icon" src={item.image_url} />
-          )}
-            <b className="book-title" >{item.title}</b>
-            <p className="book-author" >{item.author}</p>
-          </div>)}
+          <Link to={"/books/" + item.id} key={item.id}>
+            <div className="book">
+            {(item.image_url == null) ? (
```

Comment:
This logic for the cover is repeated, you can avoid it by moving it to a component.